README.md

```diff
@@ -16,7 +16,7 @@ file within the `src/apps` directory to learn how to get started.
 
 2. We have included a `.env.example` file - copy ALL these values to a `.env` file in the project root. This will ensure that you can run PillarX with no issues.
 
-3. Run `npm start` at the root. This will run PillarX locally on your machine and launch the website in your default browser.
+3. Run `npm start` at the root. This will run PillarX locally on your machine and launch the website in your default browser. Login to the wallet at http://localhost:3000/login and enter your email address - Privy will send that email address a login code.
```

Comment:
There are different ways to login, so I would suggest to change this added sentence to something like: "To access the login page, add to the url "/login". In that way, we leave the user choose how they want to login.

Reply:
Edited readme